### ISSUE TITLE
Multiverse Inventories support

### ISSUE THREAD

**Issue by Sylensky** [utarwyn/EnderContainers#81]
**Is your feature request related to a problem? Please describe.**
Actually it is related to a problem since i run 2 Gamemodes on one server and i would like to seperate inventories for those modes. Allthough i cant because only thing i can configure are disabled worlds which is currently a not so smooth workaround for it.
 
**Describe the solution you'd like**
I would like to have a multiverse inventories softdependency so it does hook into the inventory groups that distributes stuff with each other.

**Additional context**
I am sorry i don't have much to say here.

**Comment by utarwyn** [comment]
Nice idea, so it consists to support multiple enderchests for the different groups defined in the plugin **Multiverse Inventories**. Is that right? Well, in that case, a large part of **EnderContainers** must be rewritten to support multiple instances for the same enderchest, or maybe find a workaround.

**Comment by Sylensky** [comment]
Yes, i thought it would be a lot of work to make it possible. Multiverse Inventories checks if the player is in a certain group for the specific world and then allows the usage of the inventory and other shared stuff.  Atleast i couldn't think of a workaround for multiple instances right now.

**Comment by andris155** [comment]
It would be nice if you also support the Per World Inventory plugin.
https://www.spigotmc.org/resources/per-world-inventory.4482/

**Comment by TriRealm** [comment]
Hey! Andris155, I'm using this plugin for the exact same thing, and my work around is to only give users a set amount of chests for each world.

Give the user permission using your Premissions Programme to only access 
Chests 1-10 in Survival
Chests 11-20 in <Whatever other group you'd want)

Etc
![Ender Containers](https://user-images.githubusercontent.com/46031868/89147650-2e0da400-d54f-11ea-98a4-e545e8ec9a8c.JPG)

I use LuckPerms myself and all you need is these perms here give access to open certain chest only in the worlds you want

**Comment by PotatoIQ** [comment]
I hope you add support for multiverse inventories, it is a game breaking feature. Tri's suggestion is an okay solution but not an ideal one.
